src/modules/storage/migrations.ts

```diff
@@ -0,0 +1,25 @@
+import { utils } from 'decentraland-commons'
+
+export const migrations = {
+  2: (data: any) => {
+    type V1Project = { parcelLayout: { rows: number; cols: number } }
```

Comment:
Is this necessary at this stage?

Reply:
Not really but I didn't want to break any test projects. Removing it is 💯 in my opinion, but you'll need to delete your LocalStorage or the builder will explode